pkg/controller/daemon_controller.go

```diff
@@ -0,0 +1,432 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
```

Comment:
manager.go is fine, thanks.

On Wed, Jul 8, 2015 at 6:56 PM, Ananya Kumar <notifications@github.com>
wrote:

> In pkg/controller/daemon_controller.go
> <https://github.com/vmarmol/kubernetes/pull/2#discussion_r34217828>:
>
> > +Copyright 2014 The Kubernetes Authors All rights reserved.
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package controller
>
> Moved to pkg/controller/daemon/manager.go
> This file contains the DaemonManager, so I went with manager.go instead of
> controller.go. Does this seem OK?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vmarmol/kubernetes/pull/2/files#r34217828>.
>